cms/core/models/snippets.py

```diff
@@ -53,3 +58,61 @@ def save(self, *args: Any, **kwargs: Any) -> None:
 
     def __str__(self) -> str:
         return str(self.name)
+
+
+class GlossaryTerm(TranslatableMixin, PreviewableMixin, RevisionMixin, index.Indexed, models.Model):
+    """A model for glossary terms."""
+
+    title = models.CharField(max_length=255)
+    definition = RichTextField(features=settings.RICH_TEXT_BASIC)
+    # to allow for a user to be set on creation and seen in the IndexView
+    updated_by = models.ForeignKey(
+        "users.User",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="glossary_terms",
+    )
+    # See https://docs.wagtail.org/en/stable/advanced_topics/reference_index.html
+    updated_by.wagtail_reference_index_ignore = True  # type: ignore[attr-defined]
+
+    revisions = GenericRelation("wagtailcore.Revision", related_query_name="glossary_term")
+
+    panels: ClassVar[list["Panel"]] = [
+        "title",
+        "definition",
+    ]
+
+    search_fields: ClassVar[list[index.BaseField]] = [
+        *index.Indexed.search_fields,
```

Comment:
```suggestion
```

`index.Indexed` doesn't provide any search_fields. or better said `index.Indexed.search_fields == []`

Reply:
[b8bb7a7](https://github.com/ONSdigital/dis-wagtail/pull/110/commits/b8bb7a7c6b099cac41b09b2e205ba7b25d7c240c)
Note that I've also removed it from the ContactDetails snippet search fileds.